public/app/features/alerting/unified/rule-editor/formDefaults.ts

```diff
@@ -53,6 +53,7 @@ export const getDefaultFormValues = (): RuleFormValues => {
     noDataState: GrafanaAlertStateDecision.NoData,
     execErrState: GrafanaAlertStateDecision.Error,
     evaluateFor: DEFAULT_GROUP_EVALUATION_INTERVAL,
+    keepFiringFor: DEFAULT_GROUP_EVALUATION_INTERVAL, // not sure about this one
```

Comment:
I think we should default to `undefined` or `0` to disable the feature

Reply:
yes, you're right